io.github.berarma.Oversteer.yaml

```diff
@@ -0,0 +1,47 @@
+app-id: io.github.berarma.Oversteer
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+command: oversteer
+finish-args:
+  # Wayland support
+  - --socket=wayland
+  # X11 fallback support
+  - --socket=fallback-x11
+  - --share=ipc
+  - --filesystem=/run/udev
+  # Access RPM leds
+  - --filesystem=/sys/class/leds
+  # Access the steering wheel parameters
+  - --filesystem=/sys/devices
+  - --device=all
+cleanup:
+  - '/include'
+  - '/lib/pkgconfig'
+  - '/man'
+  - '/share/cmake'
+  - '/share/doc'
+  - '/share/man'
+  - '/share/vala'
+  - '*.la'
+  - '*.a'
+modules:
+  - shared-modules/lzo/lzo.json
+  - name: cairo
+    buildsystem: meson
+    sources:
+      - type: archive
+        url: https://cairographics.org/releases/cairo-1.18.0.tar.xz
```

Comment:
cairo 1.18 IS in the runtime. Why do you need this?

Reply:
You're right. I missed cairo when I was looking for applications already in the runtime. I've removed it.